package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "libxmljs2-xsd",
-  "version": "0.26.3",
+  "name": "ksys-libxmljs2-xsd",
```

Comment:
I cannot accept your PR with a name change to the library

Reply:
I will create a new branch and change the name back to the original